demo/vl-loader.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>vl-loader voorbeeld</title>
+    <script type="module" src="demo.js"></script>
+    <script type="module" src="/src/vl-loader.js"></script>
+    <script type="module" src="/node_modules/vl-ui-grid/dist/vl-grid.js"></script>
+
+    <link rel="stylesheet" type="text/css" href="/node_modules/vl-ui-core/dist/style.css" />
+    <link rel="stylesheet" type="text/css" href="/node_modules/vl-ui-grid/dist/style.css" />
+</head>
+
+<body>
+    <h1>vl-ui-loader demo</h1>
+    <p>
+        <a href="https://overheid.vlaanderen.be/webuniversum/v3/documentation/atoms/vl-ui-loader">
+            Meer voorbeelden en documentatie</a>.
+    </p>
+
+    <div>
+        <h2>Loader</h2>
+        <pre>
+            <code class="line-numbers language-markup">
+                &lt;section is="vl-region"&gt;
+                    &lt;vl-loader&gt;&lt;/vl-loader&gt;
+                &lt;/section&gt;
+            </code>
+        </pre>
+        <div class="demo">
+            <section is="vl-region">
+                <vl-loader id="loader"></vl-loader>
+            </section>
+        </div>
+    </div>
+
+    <div>
+        <h2>Loader dark</h2>
```

Comment:
De code zegt "light"

Reply:
Analogie met demo pagina van Webuniversum.

Reply:
de loader is light, de background is dark

Reply:
Bij Webuniversium staan de 3 puntjes voor de tekst, maar misschien is dat daar een bug?